diff --git a/node_modules/@jest-runner/electron/build/rpc/JestWorkerRPC.js b/node_modules/@jest-runner/electron/build/rpc/JestWorkerRPC.js
index f763e53..a39175b 100644
--- a/node_modules/@jest-runner/electron/build/rpc/JestWorkerRPC.js
+++ b/node_modules/@jest-runner/electron/build/rpc/JestWorkerRPC.js
@@ -46,7 +46,10 @@ const _runInBrowserWindow = testData => {
     const workerID = (0, _utils.makeUniqWorkerId)();
     const win = new _electron.BrowserWindow({
       show: false,
-      webPreferences: {nodeIntegration: true}
+      webPreferences: {
+        nodeIntegration: true,
+        enableRemoteModule: true
+      }
     });
 
     win.loadURL(`file://${require.resolve('../index.html')}`);
